demisto_sdk/commands/common/hook_validations/description.py

```diff
@@ -154,9 +154,9 @@ def is_duplicate_description(self):
         if not re.match(PACKS_INTEGRATION_YML_REGEX, self.file_path, re.IGNORECASE):
             package_path = os.path.dirname(self.file_path)
```

Comment:
can we change this to a `Path` object, call the constructor once, and then use it in the method?

Reply:
I dont see the point to do this, we use the `package_path` variable only when execute `self.handle_error` method which expect to get string:
`file_path (str): The file from which the error occurred`

so we need to update `handle_error` method and all the places that call it.